redesign/src/styles/layout/_grid.scss

```diff
@@ -0,0 +1,23 @@
+section.site-wrapper {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    min-height: 100vh;
+}
+
+.main {
```

Comment:
Instead of a class named "main," would you consider using the `main` element? ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main))